CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+## 0.8.4
+
+### Docker Image
+
+If you have been running a node using the v0.8.3 Docker image, the node was not saving data to the 
+specified [data directory](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#data-directory),
+or referring to the custom [configuration file](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#custom-configuration-file)
+or [genesis file](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#custom-genesis-file). 
+
+To recover the node key and data directory from the Docker container:
+`docker cp <container>:/opt/pantheon/key <destination_file>`
+`docker cp <container>:/opt/pantheon/database <destination_directory>` 
+
+Where `container` is the name or ID of the Docker container containing the Pantheon node. 
+
+The container can be running or stopped when you copy the key and data directory. If your node was 
+fully synchronized to MainNet, the data directory will be ~2TB.  
+
+When restarting your node with the v0.8.4 Docker image:
+
+* Save the node key in the [`key` file](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#node-private-key) in the data 
+    directory or specify the location using the [`--node-private-key` option](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#specifying-a-custom-node-private-key-file).  
+* Specify the `<destination_directory` as a [volume for the data directory](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#data-directory). 
+
+### Upcoming Changes in v0.9
```

Comment:
Shouldn't we put upcoming changes in a roadmap or something else ? I don't feel comfortable with having upcoming changes in a changelog. I feel that it should only list already made changes. It's just that I never saw that in other changelogs.

Reply:
@NicolasMassart These upcoming changes as by special request from @rojotek 

I think the rationale here is that they are breaking changes we have planned so Rob would like to give some forewarning.